lib/CleantalkSP/SpbctWP/State.php

```diff
@@ -43,6 +43,9 @@ class State extends \CleantalkSP\Common\State
         'bfp__delay__5_fails'               => 10,   // Delay to sleep after 5 wrong auths
         'bfp__block_period__5_fails'        => 3600, // By default ban IP for brute force for one hour
         'bfp__count_interval'               => 900,  // Counting login attempts in this interval
+        'edit_tech_support_url__enabled'    => 0,
```

Comment:
This option have to be enabled by default

Reply:
added the default value of the link

Reply:
What for? If the option is disabled, the default link is displayed

Reply:
Ok